Scripts for Variable Angle/Backtransformation_GCode_var_angle.py

```diff
@@ -1,18 +1,8 @@
 import re

```

Comment:
I don't see the change in the PR, but you probably want to make both scripts executable.

On linux or OSX:
```
chmod +x 'Scripts for Variable Angle/Backtransformation_GCode_var_angle.py'
```

No clue how to do that on windows :shrug:

Reply:
Oh, also need a shebang for unix systems
```suggestion
#!/usr/bin/env python
import re
```

Sorry for leaving each comment one-at-a-time. I'm working through it right now.

Reply:
> No clue how to do that on windows 🤷

Windows simply doesn't care about that, if the file's extension is `.py` it'll try to run it regardless of its contents and/or permissions.